packages/core-ui/src/modules/insights/utils.ts

```
@@ -62,53 +61,51 @@ export const filterChartTemplates = (chartTemplates, reportTemplates, item) => {
-    (acc, key) => (acc && acc[key] !== 'undefined' ? acc[key] : undefined),
-    obj,
+    (acc, key) => (acc && acc[key] !== "undefined" ? acc[key] : undefined),
+    obj
   );
 };
 
 export const extractData = (data: any) => {
-
-  if ((data || {}).hasOwnProperty('list')) {
-    return (data as any || {}).list || []
+  if ((data || {}).hasOwnProperty("list")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)